etc/ci/performance/test_all.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+case "$1" in
+  --servo)
+    engine=""
+    ;;
+  --gecko)
+    engine="--engine gecko"
+    ;;
+  *)
+    echo "Can't understand the argument, using the default servo engine"
+    echo "Usage: test_all.sh [--gecko]"
+    engine=""
+    ;;
+esac
+
+source venv/bin/activate
+
+echo "Starting the local server"
+python3 -m http.server > /dev/null 2>&1 &
+
+#MANIFEST="page_load_test/test.manifest"
```

Comment:
I'm guessing this is meant to be uncommented when #11087 is fixed - can you add a TODO comment with a link?